Framework/Algorithms/test/FilterByLogValueTest.h

```diff
@@ -112,8 +112,8 @@ class FilterByLogValueTest : public CxxTest::TestSuite {
                                                   // WorkspaceCreationHelper.
 
     TimeSeriesProperty<double> *temp;
-    temp = new TimeSeriesProperty<double>("temp");
-    // 10 C at 10 sec up to 50C at 50 sec
+    temp = new TimeSeriesProperty<double>("temp"); // Temperature log
+    // 10C at 10 sec up to 50C at 50 seconds
```

Comment:
Just a minor thing: in the comment, either use "sec" or "seconds" in both cases.